oauth2_http/javatests/com/google/auth/oauth2/CertificateIdentityPoolSubjectTokenSupplierTest.java

```diff
@@ -143,4 +157,274 @@ public void getSubjectToken_success() throws Exception {
     // Verify
     assertEquals(expectedSubjectToken, actualSubjectToken);
   }
+
+  @Test
+  public void getSubjectToken_trustChainWithLeaf_success() throws Exception {
```

Comment:
nit: Can this be `getSubjectToken_trustChainWithLeafFirst_success` or something else to clarify that this is only successful because the leaf is first in the trustChain. IIUC, leaf in trustchain not first is an error.